Drive/rover_drive_system.hpp

```diff
@@ -339,6 +339,18 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
       throw e;
     }
   }
+  double calculateLeftTurnAngle(double x_angle)
+  {
+        double left_angle = static_cast<double>(0.392 + 0.744 * x_angle + -0.0187 * pow(x_angle, 2) +
+        1.84E-04 * pow(x_angle, 3));
+        return left_angle;
+  }
+  double calculateBackTurnAngle(double x_angle)
```

Comment:
```suggestion
  double GetBackWheelDriveAngle(double angle)
```
[Functions are camel case](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules) starting with a capital too. I think this renaming makes it more understandable too